src/Engine/GameScenario.java

```diff
@@ -37,7 +37,7 @@ public GameEventQueue initTurn(GameMap map)
 
   public enum TagMode
   {
-    OFF(false), AWBW(true), Team_Merge(false); // Cartridge, Persistent,
+    OFF(false), AWBW(true), Persistent(true), Team_Merge(false); // Cartridge
```

Comment:
I know we don't have a UI explanation of what the tag modes are, but while we fix that, can we at least get a high-level explanation here in a comment?